Sources/PrinceOfVersions/Objective-C Helpers/ResponseModel Wrapers/AppStoreData/AppStoreUpdateResultObjectiveC.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  AppStoreUpdateResultObjectiveC.swift
+//  PrinceOfVersions
+//
+//  Created by Jasmin Abou Aldan on 13/09/2019.
+//  Copyright © 2019 Infinum Ltd. All rights reserved.
+//
+
+import Foundation
+
+@objcMembers
+public class AppStoreUpdateResultObject: NSObject {
```

Comment:
Should we hide this for swift?

Reply:
Even though this class is written to be used with Objective-C PoV methods, exposed Swift methods to ObjC are still written in Swift so we cannot hide this.